lib/mayaUsd/ufe/CMakeLists.txt

```diff
@@ -70,18 +70,18 @@ set(headers
     UsdUndoRenameCommand.h
     Utils.h
     UfeVersionCompat.h
+    UsdAttribute.h
+    UsdAttributes.h
+    UsdContextOps.h
+    UsdObject3d.h
+    UsdUndoCreateGroupCommand.h
```

Comment:
Only UsdUndoCreateGroupCommand.h seems to be V1. Everything else is V2 and should remain in the list below.

Reply:
Hamed, all of these are UFE V2 files and should remain where they are.

Reply:
Thanks @seando-adsk . Indeed. I had a momentary brain damage when I made the change :)